### DISCUSSION TITLE
[Q&A] 请问目前一共支持哪些模型？

### DISCUSSION THREAD

**Discussion by kokojacket** [lobehub/lobehub#874] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

我想通过CUSTOM_MODELS环境变了自定义模型和模型顺序，想知道目前一共支持哪些模型？

**Comment by arvinxx** [comment, marked as answer]
默认显示的是支持的，剩下你要自定义的是否可用，需要你自行保证了。

**Reply by kokojacket** [reply]
我正在使用oneapi这个项目，当我添加ERNIE-Bot-4这个模型到lobe之后，会显示"code": "model_not_found"这个错误，所以我才想问问目前支持的模型是哪些

**Reply by arvinxx** [reply]
这个感觉像是你的配置问题？

[a comment by an automated account is not shown]
